source/common/http/utility.cc

```diff
@@ -856,7 +856,7 @@ Utility::getLastAddressFromXFF(const Http::RequestHeaderMap& request_headers,
   }
 
   absl::string_view xff_string(xff_header->value().getStringView());
-  static const std::string separator(",");
+  static constexpr absl::string_view separator(",");
   // Ignore the last num_to_skip addresses at the end of XFF.
   for (uint32_t i = 0; i < num_to_skip; i++) {
     const std::string::size_type last_comma = xff_string.rfind(separator);
```

Comment:
nit: this should be absl::string_view::size_type.

Reply:
Thanks for catching this. Done.

Reply:
nit: I think there still is a `std::string::size_type` being used.

By the way, does string view contains a `absl::string_view::npos`? If it does then should we use it here?

Reply:
Thanks for catching this!
I probably should to get a new pair of eyeglasses :)
Fixed all places that referenced std::string and did not need to create an explicit string object.